src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubectlBuildWrapper.java

```diff
@@ -113,6 +112,7 @@ public void setUp(Context context, Run<?, ?> build, FilePath workspace, Launcher
     }
 
     @Extension
+    @Symbol("kubectl")
```

Comment:
```suggestion
    @Symbol("kubeconfig")
```

seems more intuitive to me